plugins/janus_videoroom.c

```diff
@@ -441,6 +442,11 @@ typedef struct janus_videoroom_rtp_relay_packet {
 
 
 /* Freeing stuff */
+static void janus_videoroom_subscriber_dereference(janus_videoroom_subscriber *s) {
+	if(s)
```

Comment:
What can lead to `s` being NULL here? According to [the documentation](https://developer.gnome.org/glib/stable/glib-Memory-Allocation.html#g-clear-pointer) `g_clear_pointer` won't call the destroy function if the passed pointer is NULL, so how is it possible to end in that situation?